src/System.Runtime.Numerics/src/System/Numerics/Complex.cs

```diff
@@ -29,20 +30,20 @@ public struct Complex : IEquatable<Complex>, IFormattable
         // This value is used inside Asin and Acos.
         private static readonly double s_log2 = Math.Log(2.0);
 
-        private double _real;
-        private double _imaginary;
+        private double m_real;
+        private double m_imaginary;
         
         public Complex(double real, double imaginary)
         {
-            _real = real;
-            _imaginary = imaginary;
+            m_real = real;
+            m_imaginary = imaginary;
```

Comment:
Maybe something inline like 
```c#
            m_real = real; // Do not rename (binary serialization)
            m_imaginary = imaginary; // Do not rename (binary serialization)
```

We already do stuff like this in types that are hard bound to the runtime eg on string and array, although I guess there will be many more like this.
Your call @morganbr